src/manifests.py

```diff
@@ -0,0 +1,88 @@
+import json
+import logging
+import traceback
+
+import yaml
+from ops.manifests import ConfigRegistry, ManifestLabel, Manifests, Patch
+
+log = logging.getLogger()
+
+
+class RemoveNamespace(Patch):
+    """Remove namespace from resources so they deploy to the local namespace"""
+
+    def __call__(self, obj):
+        obj.metadata.namespace = None
```

Comment:
any chance customers would like to config the specific namespace?  Maybe that'd be a future change?

Reply:
I think we should be okay without it. I've only ever seen users request configurable namespaces when an addon lands in the `default` namespace and they want to move it elsewhere. That won't happen here.

Removing namespaces here makes the resources land in the `<model-name>` namespace, alongside the charm itself. I think it's fair to expect charms to create resources in the namespaces they reside in - that coupling has already been established by Juju.